### PR TITLE
add create cancel actions

### PR DESCRIPTION
**Pull request by dougperez**
- Adds cancel create actions.
- Adds connectivity encryption.

### CODE REVIEW

File: lib/conduit/fortress/actions/base.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        cipher
+      end
+    end
+
+    def encode_value(value)
+      encoded_value = Base64.encode64(value).gsub("\n", "")
+      ERB::Util.url_encode(encoded_value).gsub(/%[a-f\d]{2}/i) {|s| s.downcase }
+    end
+
+    def secrete_key
```
Comment (pedromartinez) [inline lib/conduit/fortress/actions/base.rb]: secret?

File: lib/conduit/fortress/actions/base.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    def http_headers
+      {
+        "Content-Type" => "application/json",
+        "Accept" => "application/json",
+        "Authorization" => auth_token,
+        "PartnerId" => @options[:partner_id]
+      }
+    end
+
+    def attributes_with_values
```
Comment (pedromartinez) [inline lib/conduit/fortress/actions/base.rb]: Doesn't conduit have this now?

https://github.com/conduit/conduit/blob/5814581bc85262bfea71a81296863c41d097a948/lib/conduit/core/action.rb#L106

File: lib/conduit/fortress/actions/base.rb (hunk 1 above)
Comment (dougperez) [reply lib/conduit/fortress/actions/base.rb]: That just what they call it. Im up for better naming.

File: lib/conduit/fortress/actions/base.rb (hunk 2 above)
Comment (dougperez) [reply lib/conduit/fortress/actions/base.rb]: Yes, changed.

File: lib/conduit/fortress/actions/base.rb (hunk 1 above)
Comment (pedromartinez) [reply lib/conduit/fortress/actions/base.rb]: Yeah, let's call it secret_key

File: lib/conduit/fortress/actions/base.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    def cipher
+      @cipher ||= begin
+        cipher = OpenSSL::Cipher.new("des3")
+        cipher.encrypt
+        cipher.key = @options[:key]
+        cipher.iv = @options[:iv]
+        cipher
+      end
+    end
```
Comment (pedromartinez) [inline lib/conduit/fortress/actions/base.rb]: You can save some lines with tap here:

```ruby
   def cipher
    @cipher ||= OpenSSL::Cipher.new("des3").tap do |cipher|
      cipher.encrypt
      cipher.key = @options[:key]
      cipher.iv = @options[:iv]      
    end
  end
```

File: lib/conduit/fortress/actions/base.rb (hunk 1 above)
Comment (dougperez) [reply lib/conduit/fortress/actions/base.rb]: I just realized you were talking about a typo... 😿 thanks.

**Pull request merged**